.github/workflows/maven-snapshot.yml

```diff
@@ -1,4 +1,4 @@
-name: Publish package to the Maven Central Repository
+name: Publish package to the Maven Central Staging Repository
```

Comment:
nit: GitHub UI truncates long messages and shows only first half of the message. I would suggest modifying the names to "Publish Snapshot"  and "Publish Release" to easily distinguish between both actions.

Reply:
Yea good point. I will add this in the next release. 👍🏼